src/main/java/net/minecraftforge/client/model/TRSRTransformation.java

```diff
@@ -516,4 +537,54 @@ public static TRSRTransformation blockCornerToCenter(TRSRTransformation transfor
         ret.mul(tmp);
         return new TRSRTransformation(ret);
     }
+
+    @Override
+    public int hashCode()
```

Comment:
only matrix should be used for equals/hashcode, all other fields don't count for identity comparisons; don't rely too much on IDE method generators.